tests/test_triton_server.py

```diff
@@ -172,6 +173,7 @@ def test_start_stop_gpus(self):
         self.server.stop()
         self.server_local_mock.assert_server_process_terminate_called()
 
+    @skip('May not be valid')
```

Comment:
Why are you skipping this test?

Reply:
Ah I had skipped it because the logs function was not unit testable. It is tested in L0_custom_flags and L0_server_launch_modes so perhaps this can be deleted